Chapter1/Zad1.py

```diff
@@ -0,0 +1,13 @@
+'''
+Created on 23 sty 2018
+
+@author: agata.wiewiora
+'''
+# This program reads two numbers and length of the base and the height of a right-angled triangle and prints the area:
+print 'Podaj wielkosc podstawy trojkata'
+a = int(input())
+print 'Podaj wysokosc trojkata'
+h = int(input())
+pole = (a*h)/2
+print 'Pole trojkata o podstawie %i i wysokosci %i jest rowne %i' %(a,h,pole)
```

Comment:
jest nowy styl formatowania: https://pyformat.info/
można zamiast %i dać %d https://docs.python.org/2/library/stdtypes.html#string-formatting
dodaj spacje w krotce zgodnie z PEP8 https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements